fluentui_core/src/main/java/com/microsoft/fluentui/compose/ModalPopup.kt

```diff
@@ -183,75 +304,48 @@ private class ModalWindow(
                 flags
             } else flags or WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS
         }
-
-    private var content: @Composable () -> Unit by mutableStateOf({})
-
-    override var shouldCreateCompositionOnAttachedToWindow: Boolean = false
-        private set
-
-    @Composable
-    override fun Content() {
-        content()
-    }
-
-    fun setCustomContent(
-        parent: CompositionContext? = null,
-        content: @Composable () -> Unit
-    ) {
-        parent?.let { setParentCompositionContext(it) }
-        this.content = content
-        shouldCreateCompositionOnAttachedToWindow = true
     }
+}
 
-    fun show() {
-        windowManager.addView(this, params)
-    }
+@VisibleForTesting
```

Comment:
why is this annotation needed?

Reply:
I was trying with different Android versions, will remove this